app/code/Magento/InventoryInStorePickupQuote/etc/module.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryInStorePickupQuote" setup_version="1.0.0">
```

Comment:
Please get rid of setup version, it's not used any more

Reply:
Good to know that this is possible :)
Done.